### PR TITLE
Show last feed activity in profile

### PR DESCRIPTION
**Pull request by black-puppydog**
Brought up by [@kas](@RuNxm8SRujPcJx6GjtTQHp6hprAFv5voEkcvoAkB8Pk=.ed25519) in [`%j0/mywlnDar3HJ6ED34ICxzyep3QhM0hvi1ov12A04s=.sha256`](https://ssb.muchmuch.coffee/%25j0%2FmywlnDar3HJ6ED34ICxzyep3QhM0hvi1ov12A04s%3D.sha256). This shows the timestamp of the last message received from a feed.

Not sure thought if I did this the right way, depject is still quite the mystery to me...
I *think* I got the data out of sbot the right way, but not about whether I hooked it up to mutant correctly.

Also not sure about the positioning in the profile view. I put it below the avatar. Always felt like that space was very empty, and this seems to fit in the quite nicely.
Anyhow, another learning experience :P

### CODE REVIEW

**Comment by black-puppydog** [comment]
@christianbundy what's happening up there? :point_up_2: 
I was expecting Travis, but instead I got this sonarcloud thingy that doesn't seem to really run our code, and which thinks we have 0% code duplication, which I find hard to believe.

**Comment by black-puppydog** [comment]
okay nevermind seems to have been a github page refresh problem... :thinking:

**Comment by christianbundy** [comment]
Damn it. I was testing out an app and gave it permission to look at my repos, I didn't consent to it spamming a bunch of bullshit on every pull request. Sorry about that, I'm reporting the spam and blocking the service from my account.

File: lib/depject/page/html/render/profile.js [hunk 1]
```diff
@@ -142,7 +154,10 @@ exports.create = function (api) {
     ])
 
     const prepend = h('header', { className: 'ProfileHeader' }, [
-      h('div.image', api.about.html.image(id)),
+      h('div.image', [
+        api.about.html.image(id),
+        h('div.lastActivity', computed(lastActivity, msg => `Last activity: ${api.message.html.timestamp(msg, false)}`))
```
Comment (Powersource) [inline lib/depject/page/html/render/profile.js:159]: Add translate-ability to "Last activity" like this https://github.com/ssbc/patchwork/blob/33bda27b8cca77badcd0c336d1d6bf580c8ff077/lib/depject/page/html/render/settings.js#L48

**Comment by Powersource** [comment]
Can we have a screenshot? (reviewing from a distance :D )

And I'm a bit with hoodownr's worry (if I remember the thread correctly) about staring at timestamps. How granular is the timestamp atm?

**Comment by black-puppydog** [comment]
image
I just put it down there under the profile picture. Not sure it's the best place, honestly, but I didn't want to make the profile page even longer, pushing the feed contents further down. Not married to this placement though.

This is using [moment-timezone](https://www.npmjs.com/package/moment-timezone) so it will show "relevant" timestamps like "1 sec ago" or "5 hours ago".
So yes, I can see that concern. I'll have a look into that.

**Comment by christianbundy** [comment]
Looks great to me, thanks for this patch! I'd be happy to merge any improvements if anyone wants something different.

**Comment by Powersource** [comment]
I can understand that you want to merge improvements sooner rather than later Christian. But I don't understand why you merged this already, we were just in the middle of a discussion on whether this was overall harmful for patchwork users. 

I can see how this can get healthy with some more work but atm I'd vote for reverting this.

**Comment by christianbundy** [comment]
Sorry, it wasn't clear to me that you had a blocking concern. If you propose a revert I'd be happy to merge that.

**Comment by Powersource** [comment]
Follow-up pr if people are curious https://github.com/ssbc/patchwork/pull/1290

**Pull request merged**